substrate/frame/staking-async/src/lib.rs

```diff
@@ -155,18 +156,16 @@ pub struct ActiveEraInfo {
 /// Reward points of an era. Used to split era total payout between validators.
 ///
 /// This points will be used to reward validators and their respective nominators.
-#[derive(PartialEq, Encode, Decode, RuntimeDebug, TypeInfo)]
-pub struct EraRewardPoints<AccountId: Ord> {
+#[derive(
+	PartialEqNoBound, Encode, Decode, DebugNoBound, TypeInfo, MaxEncodedLen, DefaultNoBound,
+)]
+#[codec(mel_bound())]
+#[scale_info(skip_type_params(T))]
+pub struct EraRewardPoints<T: Config> {
```

Comment:
Pulling in `Config` is a nightmare for anyone using it. Please try to only pull in `Config` if absolutely necessary.  
It is better here to just pull in what is needed: `AccountID` and the MEL bound.